react-components/src/architecture/base/concreteCommands/NavigationTool.ts

```diff
@@ -33,10 +33,14 @@ export class NavigationTool extends BaseTool {
     return { key: 'NAVIGATION', fallback: 'Navigation' };
   }
 
-  public override onHover(event: PointerEvent): void {
+  public override onHoverByDebounce(event: PointerEvent): void {
     this.renderTarget.viewer.onHover360Images(event);
   }
 
+  public override onHover(event: MouseEvent): void {
+    this.renderTarget.viewer.onHover360Images(event as PointerEvent);
+  }
```

Comment:
Should we call `onHover360Images` in both methods?

Reply:
Also - should we fix the `PointerEvent` <-> `MouseEvent` issue in the function signature?

Reply:
It should be MouseEvent. It is the old one that should have MouseEvent